test-lib-openshift.sh

```diff
@@ -316,10 +316,11 @@ function ct_os_cluster_up() {
   local cluster_ip="127.0.0.1"
   [ "${is_public}" == "true" ] && cluster_ip=$(ct_get_public_ip)
 
+  ct_os_set_path_oc "${cluster_version}"
```

Comment:
What if the `cluster_version` is not set?

Reply:
Good point, we need to have some version every-time, but rather than hard-code e.g. latest released, I'd expected we should simply use what we have installed. What do you think?